vignettes/cqc_directory.Rmd

```diff
@@ -0,0 +1,146 @@
+---
+title: "cqc_directory dataset"
+author:
+- '[Martine Wauben](mailto:martine.wauben@dhsc.gov.uk)'
+date: "04/10/2020"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{cqc_directory dataset}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+options(tidyverse.quiet = TRUE)
+```
+
+This vignette explains how to use the `cqc_directory` dataset in R, and also
+details where it comes from and how it is generated.
+
+The data is sourced from [the Care Quality Commission](https://www.cqc.org.uk/about-us/transparency/using-cqc-data)
+and its archive is available on [Google Drive](https://drive.google.com/drive/folders/1Y6V6r-q2l4lJYKuZL0DXw25VDUTH6L4N). 
+
+The data contains care provider locations and ratings as published monthly in 2019. 
+
+The dataset contains data on locations and providers:
+
+- __location__: location ID, name, ODS code, telephone number, web address, sector, inspection directorate, region, local authority, CCG, street address, parliamentary constituency, and coordinates.
+- __provider__: provider ID, name, company/charity registration, telephone number, web address, sector, inspection directorate, region, local authority, street address, and parliamentary constituency.
+code for the organisation that this activity relates to
+- __pub_date__: The date that particular datapoint was published
+
+First let's load some packages and the dataset and show the first 10 rows of data.
+
+```{r load data, message=TRUE, warning=TRUE}
+library(dplyr)
+library(knitr)
+library(tidyr)
+library(ggplot2)
+library(lubridate)
+
+library(NHSRdatasets)
+
+data("cqc_directory")
```

Comment:
this is failing to run - the file cqc_directory.Rda should have single item called "cqc_directory", it looks like this is loading an object called cqc_load however